core/ui/prime/components/src/lib/table/table.component.html

```diff
@@ -9,11 +9,15 @@
     <ng-template pTemplate="header">
         <tr>
             <th *ngFor="let column of columns.columns">{{ column.label }}</th>
+            <th *ngIf="hasRowActions()" style="width: 32px"></th>
         </tr>
     </ng-template>
     <ng-template pTemplate="body" let-item>
-        <tr>
+        <tr (click)="onRowClicked(item)">
             <td *ngFor="let column of columns.columns">{{ item[column.field] }}</td>
+            <td *ngIf="hasRowActions()"><lens-button icon="pi-ellipsis-v" appearance="text" size="small" (clicked)="onRowActionButtonClicked(menu, $event, item)"></lens-button></td>
```

Comment:
Why is this fixed? Can't this be just another column of type action-column? So, the rendering of the column is different based on the type, but you leave it up to the user of the table to decide where the column should appear. Could even be multiple columns.